packages/vats/test/bootstrapTests/test-vaults-integration.js

```diff
@@ -288,3 +288,69 @@ test('exit bid', async t => {
     },
   });
 });
+
+test('propose change to auction governance param', async t => {
+  const { walletFactoryDriver, agoricNamesRemotes, storage } = t.context;
+
+  const gov1 = 'agoric1ldmtatp24qlllgxmrsjzcpe20fvlkp448zcuce';
+  const wd = await walletFactoryDriver.provideSmartWallet(gov1);
+
+  t.log('accept charter invitation');
+  {
+    const instance = agoricNamesRemotes.instance.econCommitteeCharter;
```

Comment:
not good to have two variables with the same name and overlapping scopes. Can this be `committee` and the other be `auctioneer`?

Reply:
The scopes don't overlap. That's the purpose of the `{` on 299: to introduce a new scope so that I don't have to think of novel names.

But ok, since you supplied novel names, will do.

Reply:
I can no longer see the code (I thought github was supposed to make it straightforward to look at older versions of a PR) so I can no longer prove it, but I thought what I saw was the second scope was fully contained within the first, which is the "not good" form of overlap I was talking about. Did I mis-read it? Were the scopes fully non-overlapping?